streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractTask.java

```diff
@@ -251,24 +246,4 @@ public boolean hasStateStores() {
         return stateMgr.changelogPartitions();
     }
 
-    Map<TopicPartition, Long> committedOffsetForPartitions(final Set<TopicPartition> partitions) {
```

Comment:
This is moved into `StandbyTask` as we don't share it any longer for both types of tasks.